examples/do-not-use/docs/aws-load-balancer-controller.md

```diff
@@ -0,0 +1,86 @@
+# AWS Load Balancer Controller.
+
+[AWS Load Balancer Controller ](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.5/) is a controller to help manage Elastic Load Balancers for a Kubernetes cluster. This Add-on deploys this controller in an Amazon EKS Cluster.
+
+## Usage 
+
+In order to deploy the AWS Load Balancer Controller Addon via [EKS Blueprints Addons](https://github.com/aws-ia/terraform-aws-eks-blueprints-addons), reference the following parameters under the `module.eks_blueprints_addons` as shown in this [example](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/examples/fargate-serverless/main.tf#L144-L156).
```

Comment:
Remove all references back to examples - examples can change and move and this will break any of theses links

Reply:
Done!